views/faqs/index.js

```diff
@@ -35,6 +35,11 @@ export class FaqView extends React.Component {
       tracker.trackException(err.message)
       console.warn(err.message)
     }
+
+    if (__DEV__) {
+      html = faqs
```

Comment:
It's using the text imported from the json above.

`import {text as faqs} from '../../docs/faqs.json'`